docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc

```diff
@@ -0,0 +1,131 @@
+* If you click **Open Git repository**, a new tab opens to the GitHub repository for the code framework you chose.
+* If you click **Import sample**, {ProductName} creates and deploys a new application with the sample code you chose. Check out the open **Overview** tab to monitor your continuous integration/continuous deployment (CI/CD) activity.
+
+.What's next
+**Configure your components for deployment** opens. From here, configure your components before you deploy them. You can also review or change any values that {ProductName} populated for you. When you're finished here, you're one click away from creating your application.
+
+//// 
+From JSese: May 15, 2023, Commenting out per Matt Reid and Christian V. Support for private repos is post-Summit.
+[discrete]
+== Granting App Studio access to your Git repository
+
+If {ProductName} can’t access your Git repository, the following message displays: “We can’t access your repository, probably because it’s either private or the URL is incorrect.” If your repository is private, choose one of the following two ways to allow {ProductName} to interact with your repository:
+
+[discrete]
+=== Granting App Studio access to your Git repository through GitHub OAuth
+
+. Under **Authorization**, click **Sign in**. **GitHub Authorize OAuth** opens in a new tab.
+. Click **Authorize _redhat-appstudio_**.
+
+[discrete]
+=== Granting App Studio access to your Git repository with a token
+
+. Under **Authorization**, click **Use a token instead**. The **Authenticate with API token** modal opens.
+. Enter your **Username**.
+. Enter an **API token**.
+. Click **Connect**.
+////
+
+//[id="proc_reviewing_configging_components_{context}"]
+
+== Reviewing and configuring components
+
+[role="_abstract"]
+If you imported your own code, from the **Review and configure for deployment** pane, make sure to define, review, or change your application and component settings before you finish importing. You can rename a component here, too. 
+
+. Make sure that **Component name** is correct. Click the GitHub link under this field if you want to check your repository URL.
+. Confirm that {ProductName} detected the right **Runtime**. If so, you should notice that the suggested deployment options are appropriate. If you want to change your runtime, expand the menu next to your runtime name and make a selection.
+
+[IMPORTANT]
+====
+If you change the runtime, your component builds and deploys using the runtime devfile instead of the component devfile or Dockerfile.
+====
+
+In the **Build & deploy configuration** section, look over these fields:
+
+[start=3]
+. **Target port**: Is it correct? If not, click in the field to modify it.
+. **Dockerfile**: If you specified **Dockerfile** as your runtime, make sure {ProductName} detected the right one. Click in the field if you need to modify it. Don't worry! {ProductName} hides this field if you chose a different runtime.
```

Comment:
> Don't worry! {ProductName} hides this field if you chose a different runtime.

This doesn't actually happen. But I do seem to recall that as part of the ask. I'll raise a bug for the UI to investigate this.

Reply:
thanks so much, @christianvogt. will it change by Summit?

Reply:
Not likely. At this point nothing but blockers will be going in.

Reply:
ok thanks. i'll hide until further notice (UFN)

Reply:
Yeah, the dockerfile field behavior won't be changing in the next week or two from how it acts today (where it always shows).

Reply:
ok thanks, guys